app/scripts.babel/locales.js

```diff
@@ -0,0 +1,21 @@
+// We can't use messages.json for these messages because the locale for them
+// depends on the site content
+const MESSAGES = {
+  en: {
+    friends_ranklist: 'Friends ranklist',
+    follow: 'Add to friends',
+    unfollow: 'Remove from friends',
+    no_friends: 'Use the $star$ icon to add friends.',
+  },
+  ru: {
+    friends_ranklist: 'Рейтинг друзей',
+    follow: 'Добавить в друзья',
+    unfollow: 'Удалить из друзей',
+    no_friends: 'Используйте кнопку $star$, чтобы добавить друзей.',
+  },
+};
+
+function getSiteMessages(callback) {
+  observer.forEach('.panel a[href="/news.aspx"]', newsLink =>
+    callback(MESSAGES[newsLink.innerText === 'Site news' ? 'en' : 'ru']));
```

Comment:
This does not always work for me:
![screen shot 2016-07-10 at 5 22 59 pm](https://cloud.githubusercontent.com/assets/470136/16717105/bd0c2fec-46c3-11e6-9804-66b2b9bb880b.png)
I think `newsLink.innerText` is not always ready at this time.